metrics/torch_is_fid_score.py

```diff
@@ -9,18 +9,18 @@
 
 from metrics.torch_inception import InceptionV3
 from metrics.torch_fid_utils import calculate_frechet_distance, torch_cov
-from utils.utils import check_gpu
+from utils.utils import check_gpu, set_seed
 
 device = check_gpu()
 
+# Randomly generate the same set of z vectors each time
+set_seed(42)
```

Comment:
Setting the seed outside. 
Also the fn `calc_is_and_fid ` requires  torch.backends.cudnn.deterministic = True
which `set_seed` does